templates/component-mapper/demo/demo-schemas/miq-schema.js

```diff
@@ -0,0 +1,1949 @@
+/* eslint-disable camelcase */
+const input = {
```

Comment:
I don't think it's necessary to copy this schema. I think we can even remove it completely